README.md

```diff
@@ -2,16 +2,14 @@
 
 [![https://www.npmjs.com Package](https://github.com/TIGNUM/pdf-generation-actions-web-component/actions/workflows/npm-publish-package.yml/badge.svg?branch=main&event=workflow_run)](https://github.com/TIGNUM/pdf-generation-actions-web-component/actions/workflows/npm-publish-package.yml)
 
-Print or pdf generation buttons for generating Tignum reports
+Print or generate a pdf from the html passed as string to the Web Component
```

Comment:
We need to include this is only available for moder browsers because we are not polifilling for IE11

Reply:
Modified